ocs_ci/ocs/resources/pod.py

```diff
@@ -2801,6 +2803,43 @@ def check_toleration_on_subscriptions(toleration_key=constants.TOLERATION_KEY):
         )
 
 
+def check_duplicate_tolerations_on_pods():
+    """
+    Function to check for duplicate tolerations on pods.
```

Comment:
you are checking for all the pods in the openshift-storage namespace. Please update the description of the function accordingly.